project.yml

```diff
@@ -138,3 +141,25 @@ targets:
         excludes: Info.plist
         createIntermediateGroups: true
 
+  # UITests1
+
+  GoldenSampleUITests:
+    type: bundle.ui-testing
+    platform: iOS
+    info:
+      path: appUITests/Info.plist
+    codeSign: false
+    deploymentTarget:
+      iOS: 15.0
```

Comment:
Why the oldest one? Yes, we are expected to support 15-17, just curious why that one.